Python/test_isvc.py

```diff
@@ -5,24 +5,78 @@
 import utils
 from isvc import iSVC
 import numpy as np
+import isvc_gui as gui
```

Comment:
The error might be due to here. After you upgraded your matplotlib, maybe you can have a look at it.